terraform/modules/network-validator/scripts/network_validation.sh

```diff
@@ -512,11 +514,14 @@ This script is used to validate existing subnets, and optionally network securit
   -p, --http_port     WebLogic Admin Console http port (defaults to 7001)
   -s, --https_port    WebLogic Admin Console https port (defaults to 7002)
   -d, --ocidbid       OCI Database System OCID
+  -P, --ocidbport     OCI Database Port
   -t, --atpdbid       ATP Database OCID
   -g, --lpg           OCID of the Local Peering Gateway (LPG) in the DB VCN
   -b, --bastionsubnet Bastion Subnet OCID
-  -i, --bastionip     Bastion Host IP. Provide this if using existing bastion or new bastion with reserved IP
-  -l, --lbsubnet      Load Balancer Subnet OCID
+  -i, --bastionip     Bastion Host IP. Provide this if using existing bastion
+  -u, --lbsubnet1    Load Balancer Subnet 1 OCID
+  -v, --lbsubnet2    Load Balancer Subnet 2 OCID
```

Comment:
We can mention that "Load Balancer Subnet 2 OCID" is required only for AD subnet.

Reply:
Fixed